tests/test_cases/test_package/test_package.py

```diff
@@ -6,8 +6,27 @@
 A set of tests that demonstrate package access
 """
 
+import os
+
 import cocotb
-from cocotb.handle import HierarchyObject, LogicArrayObject
+from cocotb.handle import (
+    HierarchyObject,
+    IntegerObject,
+    LogicArrayObject,
+    StringObject,
+)
+from cocotb_tools.sim_versions import NvcVersion
+
+SIM_NAME = cocotb.SIM_NAME.lower()
+LANGUAGE = os.environ["TOPLEVEL_LANG"].lower().strip()
+
+questa_vhpi = (
+    SIM_NAME.startswith("modelsim") and os.getenv("VHDL_GPI_INTERFACE", "fli") == "vhpi"
+)
+
+nvc_pre_1_16 = SIM_NAME.startswith("nvc") and (
+    NvcVersion(cocotb.SIM_VERSION) < NvcVersion("1.16")
+)
```

Comment:
@nickg If you think NVC will do a 1.15.3 release with the fix, please update this.

Reply:
I'm not going to backport this so 1.16 is correct.